boards/arm/frdm_k64f/Makefile.board

```diff
@@ -4,4 +4,6 @@ DEBUG_SCRIPT = openocd.sh
 OPENOCD_LOAD_CMD = "flash write_image erase ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
 OPENOCD_VERIFY_CMD = "verify_image ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
 
-export FLASH_SCRIPT OPENOCD_LOAD_CMD OPENOCD_VERIFY_CMD
+JLINK_DEVICE = MK64FN1M0xxx12
```

Comment:
we need to add support for different flashing methods via 'make flash', right now only one is supported...